app/assets/stylesheets/employees/form.scss

```diff
@@ -0,0 +1,14 @@
+.employee_form{
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: space-around;
```

Comment:
it is good development practice to alphabetize styles

Reply:
Ok. I just have a doubt, "align items" is a flex property. Should I put this before or after "display:flex"?.

Reply:
change added